docs/modules/operation/pages/deep-dive/admin/logging/log-file-viewer.adoc

```diff
@@ -2,20 +2,19 @@
 [[ga-operation-log-file-viewer]]
 = View Log Files
 
-You can view log files using the {page-component-title} UI.
+You can view log files in the {page-component-title} UI.
+To do so, click menu:Info[Logs] in the top menu bar.
 
-Navigate to the new UI (click *UI Preview*), and select "Logs" from the navigation pane.
-The Log File Viewer is displayed.
-
-.Log File Viewer
+.Log File Viewer page
 image::logging/log-file-viewer.png[]
```

Comment:
```suggestion
image::logging/log-file-viewer.png[log file viewer, 300]
```
I might have the syntax wrong, but something to make the displayed image smaller.